.github/workflows/codeql.yml

```diff
@@ -1,76 +0,0 @@
-# For most projects, this workflow file will not need changing; you simply need
```

Comment:
this is intended?

Reply:
Yes, the codeql is not working properly, fixing it should be in another pr, so i just remove it. Or maybe we just remove it, since hte monorepo as a working version.